### PR TITLE
Laboratory work #4, Alina Batrasova - 23FPL1

### PR DESCRIPTION
**Pull request by AlinaBatrasova**

### CODE REVIEW

**Comment by Vasilisa-Blyudova** [comment]
No progress!

File: lab_4_retrieval_w_clustering/main.py [hunk 1]
```diff
@@ -26,7 +27,9 @@ def get_paragraphs(text: str) -> list[str]:
     Returns:
         list[str]: Paragraphs from document.
     """
-
+    if not text or not isinstance(text,str):
```
Comment (Vasilisa-Blyudova) [inline lab_4_retrieval_w_clustering/main.py:30]: add whitespace befor `str`

**Comment by Vasilisa-Blyudova** [comment]
no changes since last review

**Comment by Vasilisa-Blyudova** [comment]
No changes since last review

File: lab_4_retrieval_w_clustering/main.py (hunk 1 above)
Comment (AlinaBatrasova) [reply lab_4_retrieval_w_clustering/main.py:31]: done

**Pull request closed without merging**